tests/unit/TransitionWithPresentationTests.swift

```diff
@@ -57,27 +55,103 @@ class TransitionWithPresentationTests: XCTestCase {
 
     waitForExpectations(timeout: 0.5)
 
-    XCTAssert(presentedViewController.presentationController is TransitionPresentationController)
+    XCTAssert(presentedViewController.presentationController is TestingPresentationController)
+  }
+
+  func testPresentedFrameMatchesWindowFrame() {
+    let presentedViewController = UIViewController()
+    let transition = InstantCompletionTransition()
+    presentedViewController.transitionController.transition = transition
+
+    let didComplete = expectation(description: "Did complete")
+    window.frame = CGRect(x: 0, y: 0, width: 300, height: 200)
+    window.rootViewController!.present(presentedViewController, animated: true) {
+      didComplete.fulfill()
+    }
+
+    waitForExpectations(timeout: 0.1)
+
+    XCTAssertEqual(window.rootViewController!.presentedViewController, presentedViewController)
+    XCTAssertEqual(window.rootViewController!.presentedViewController?.view.bounds, window.bounds)
```

Comment:
Does make sense to check the view's frame against the window's bounds?  Also, this test method name should be updated to reflect frame/bounds.

Reply:
Done.